astropy/timeseries/periodograms/lombscargle/implementations/fast_impl.py

```diff
@@ -41,8 +42,15 @@ def lombscargle_fast(
         the result. Otherwise, use a slower O[N^2] algorithm
     trig_sum_kwds : dict or None, optional
         extra keyword arguments to pass to the ``trig_sum`` utility.
-        Options are ``oversampling`` and ``Mfft``. See documentation
+        Options are ``oversampling``, ``Mfft`` and ``eps``. See documentation
         of ``trig_sum`` for details.
+    algorithm : str, optional
+        Referenced only if use_fft is true.
+        Specify the approximation used to approximate the NUDFT of type 1. If the value is not valid falls back to the default option.
+        Currently there are two available options:
```

Comment:
let's rephrase this in a future-compatible way so it doesn't risk going out of sync
```suggestion
        Supported options are:
```